File: Java/DayTwo/GroceryApp.java
import java.util.Scanner;

public class GroceryApp{
public static void main(String[] args){
Scanner input = new Scanner(System.in);
String[] myLists = new String[10];

String menu = """
    WELCOME TO ALIMZY GROCERY APP
1. Add Items
2. Remove Items
3. Show all available items
4. Exit

""";
System.out.println(menu);

System.out.println("Enter an options sir/ma");
int menuCase = input.nextInt();

switch(menuCase){
    case 1->{
       System.out.println("How many items do you want to add sir/ ma: ");
       int addNumber = input.nextInt();
        
       for(int count = 0; count <= addNumber;count++){
        System.out.println("Enter an item to add sir/ ma: ");
        String addItems = input.nextLine();
        
        myLists[count] += addItems;
        System.out.println("Added successful");
        
}
    case 2->{
        System.out.println("How many items do you want to remove sir/ ma: ");
        int removeNumber = input.nextInt();
        for(int count = 0;count <= removeNumber; count++){
        System.out.println("Enter an item you want to remove sir/ ma");
        
        } 
        

        


}
      
        


        }
        





}

}

}

File: Python/DayTwo/GroceryApp.py
my_lists = []
while True:

    menu = """
        WELCOME TO ALIMZY GROCERY APP
    1. Add Items
    2. Remove Items
    3. show all available item
    4. Exit
    """
    print(menu)

    menu = int(input("Enter options: "))
    
    
    match menu:
        case 1:
            input_number = int(input("How many items do you want to add sir/ma: "))
            for items in range(input_number):           
                add_input = input("Add items sir/ ma: ").lower()             
                my_lists.append(add_input)
            print("Item succesfully Added")
        case 2:           
            input_number = int(input("How many items do you want to remove sir/ ma"))
            for items in range(input_number):
                remove_book = input("Remove items sir/ ma:").lower()
                if remove_book in my_lists:                               
                    my_lists.remove(remove_book)
                print("items succesfully removed")
                else:
                    print("cannot find grocery list sir/ma. Check again!")
                
        case 3:           
            print(my_lists)
        case 4:
            break
                 
    


